qc/arabidopsis_thaliana_qc.py

```diff
@@ -1,92 +1,83 @@
-import msprime
-import numpy as np
-import stdpopsim.models as models
-
-
-class Durvasula2017MSMC(models.Model):

```

Comment:
To avoid changing too much code at once, we could leave the QC module as it is and refactor it as a follow up. This would give us some confidence that we're not making any model-breaking changes in the diff.